### PR TITLE
Add type definition for test helper import

### PR DESCRIPTION
**Pull request by chriskrycho**
Resolves typed-ember/ember-cli-typescript#983

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Using the same "brand" as the types for `htmlbars-inline-precompile` for
+// backwards compatibility. The actual value of the brand doesn't matter; it is
+// only important that it (a) is distinct and (b) interoperates with existing
+// uses of the `hbs` export from `htmlbars-inline-precompile`.
```
Comment (rwjblue) [inline index.d.ts]: Can you link to the "source of truth" for this brand in this comment?

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ *   setupRenderingTest(hooks);
+ *
+ *   test('you can render things', function(assert) {
+ *     await render(hbs`<TestingComponents @isCool={{true}} />`);
+ *     assert.ok(true);
+ *   });
+ * });
+ * ```
+ */
+export function hbs(tagged: TemplateStringsArray): TemplateFactory;
```
Comment (rwjblue) [inline index.d.ts]: We support additional invocation like:

```suggestion
interface PrecompileOptions {
  moduleName?: string;

  parseOptions?: {
    srcName?: string;
  }
}

export function hbs(template: string, options?: PrecompileOptions): TemplateFactory;
export function hbs(tagged: TemplateStringsArray): TemplateFactory;
```

File: tests/dummy/lib/module-name-inliner/node_modules/ember-cli-version-checker/node_modules/.bin/semver [hunk 3]
```diff
@@ -0,0 +1 @@
+../../../../../../../../node_modules/semver/bin/semver.js
```
Comment (rwjblue) [inline tests/dummy/lib/module-name-inliner/node_modules/ember-cli-version-checker/node_modules/.bin/semver]: O_o

File: index.d.ts (hunk 1 above)
Comment (chriskrycho) [reply index.d.ts]: Excellent suggestion; will do.

File: index.d.ts (hunk 2 above)
Comment (chriskrycho) [reply index.d.ts]: TIL!

File: tests/dummy/lib/module-name-inliner/node_modules/ember-cli-version-checker/node_modules/.bin/semver (hunk 3 above)
Comment (chriskrycho) [reply tests/dummy/lib/module-name-inliner/node_modules/ember-cli-version-checker/node_modules/.bin/semver]: Ummm. What.

File: index.d.ts (hunk 2 above)
Comment (chriskrycho) [reply index.d.ts]: Done in 99738f2

File: index.d.ts (hunk 1 above)
Comment (chriskrycho) [reply index.d.ts]: Done in 99738f2

File: tests/dummy/lib/module-name-inliner/node_modules/ember-cli-version-checker/node_modules/.bin/semver (hunk 3 above)
Comment (chriskrycho) [reply tests/dummy/lib/module-name-inliner/node_modules/ember-cli-version-checker/node_modules/.bin/semver]: Removed in 4fe957e. No idea what that was about.

**Comment by richard-viney** [comment]
In testing `ember-cli-htmlbars@4.3.0` it doesn't appear to actually include the new type definitions in the package, i.e. `node_modules/ember-cli-htmlbars/index.d.ts` doesn't exist after installing with `yarn`.

Inspecting the contents of `https://registry.npmjs.org/ember-cli-htmlbars/-/ember-cli-htmlbars-4.3.0.tgz`  also shows no type definitions are present.

**Comment by chriskrycho** [comment]
Doh! That would be because the `files` key in package.json only specifies `lib`, I believe. Should be an easy and quick fix; I’ll have a PR up in an hour or so.

**Comment by rwjblue** [comment]
Fixed in #494

**Comment by chriskrycho** [comment]
SNIPED!

**Pull request merged**